plugins/dopf/core/dopf.pas

```diff
@@ -360,6 +360,7 @@   generic TdGOpf<T1, T2, T3> = class(TdComponent)
     FOnUpdated: TNotifyEvent;
     FOnUpdating: TNotifyEvent;
     FConnection: T1;
+    FFieldQuote: AnsiChar;
```

Comment:
Regarding `string` instead of `Char` - I was worried that this would cause unnecessary overhead due to the use of a string type variable where presumably only one character would be used. So I would change to Char. But I would like to hear your thoughts why to use string.

Yes, I think we can change the case of these expressions. I agree

Reply:
I'm not sure that using `Char` instead of `String` gives a significant performance gain, but with string this pullrequest will be more concise. I'm in doubt

Reply:
In this case, using `string` it's OK, because the small overhead is only at assignment time, i.e. possibly once when we pass the quote (character) to the field (property) after some object fill. Using `char` maybe add a new call (`fpc_char_to_ansistr()`) to the stack to convert the char to string at concatenation time, i.e. every time a SQL is generated. Please take a look at the following example and their respective generated asm:

Using `Char`:
```pascal
var
  S: string;
  C: Char;
  X: string;
begin
  S := 'ab';
  C := 'c';
  X := C + S;
--- asm ---
X := C + S;
movzbl 0x2f770(%rip), %esi
lea -0x68(%rbp), %rdi
mov $0x0, %edx
callq fpc_char_to_ansistr
mov -0x68(%rbp), %rsi
mov 0x2f747(%rip), %rdx
lea 0x2f760(%rip), %rdi
mov $0x0, %ecx
callq fpc_ansistr_concat
```
Using `string`:
```pascal
var
  S: string;
  C: string;
  X: string;
begin
  S := 'ab';
  C := 'c';
  X := C + S;
--- asm ---
X := C + S;
mov 0x2f79e(%rip), %rdx
mov 0x2f7a7(%rip), %rsi
lea 0x2f7b0(%rip), %rdi
mov $0x0, %ecx
callq fpc_ansistr_concat
```

Reply:
I have understood. Thanks